client/src/components/v3/StatisticRow.module.css

```diff
@@ -0,0 +1,29 @@
+.statContainer {
+  display: flex;
+  margin-top: 25px;
+  margin-bottom: 30px;
+  margin-left: -8px;
+}
+
+.statSpeed {
+  margin-left: 8px;
+  margin-right: 8px;
+  width: 115px;
+}
+
+.statPower {
+  margin-left: 8px;
+  margin-right: 8px;
+  width: 70px;
+}
+
+.statHeartRate {
+  margin-left: 15px;
+  margin-right: 8px;
+  width: 85px;
+}
```

Comment:
It would be really nice if all of these could share the same CSS, but I assume you've already tried that and have a reason

Reply:
Power, Speed, and HearRate each have different widths due to the size of the unit label. Could make the left and right margins into a class and have two classnames for each statistic, but i thought that wouldn't be that necessary.

Reply:
I was thinking maybe these widths could be completed automatically. (I think we could possibly use a flexbox with even spacing?)

Reply:
That would be difficult as the width would then change according to the stat description. Currently, a width of 115px allows the trap speed descriptions to be wrapped but making it automatic would make it look too wide.
Also, the widths of the stats would keep changing due to the widths of the numbers which may be a bit jarring to look at.

If there is a particular implementation you've found to work well, I'm happy to go with it.

Reply:
Don't worry about this for now. Might experiment with it myself later